modules/images/regenerate-existing-images/load.php

```diff
@@ -7,3 +7,79 @@
  * @since n.e.x.t
  * @package performance-lab
  */
+
+/**
+ * Register the background job taxonomy.
+ *
+ * Intentionally on lower priority, so that other post types can be
+ * registered successfully and taxonomy can be used for all the post types.
+ *
+ * This taxonomy will be used as a queue system where each term represents a job in the queue.
+ *
+ * @return void
+ */
+function perflab_background_job_taxonomy() {
```

Comment:
Let's rename this to a more "descriptive" `perflab_register_background_job_taxonomy()`. Function names should typically begin with an action verb.